modules/apps/object/object-test/src/testIntegration/java/com/liferay/object/web/internal/object/entries/display/context/test/ObjectEntryDisplayContextTest.java

```diff
@@ -478,44 +493,105 @@ private MockHttpServletRequest _getMockHttpServletRequest(
 
 		return _getMockHttpServletRequest(
 			externalReferenceCode, LocaleUtil.getDefault(), objectDefinition,
-			objectRelationshipId, parentObjectEntryERC);
+			objectRelationshipId, parentObjectEntryERC, _group.getGroupId());
 	}
 
 	private void _testGetAPIURL(
 			ObjectEntry objectEntry, ObjectRelationship objectRelationship,
 			ObjectEntry parentObjectEntry)
 		throws Exception {
 
-		ObjectDefinition objectDefinition =
-			_objectDefinitionLocalService.getObjectDefinition(
-				objectRelationship.getObjectDefinitionId1());
+		MockHttpServletRequest mockHttpServletRequest;
+
+		String apiURL;
 
-		String apiURL = StringBundler.concat(
-			"/o", objectDefinition.getRESTContextPath(),
-			"/by-external-reference-code/",
-			parentObjectEntry.getExternalReferenceCode(), "/",
-			objectRelationship.getName());
+		ObjectDefinition objectDefinition;
 
-		if (Objects.equals(
-				objectDefinition.getScope(),
-				ObjectDefinitionConstants.SCOPE_SITE)) {
+		boolean hasRelationShip = false;
+
+		if (objectRelationship != null) {
+			hasRelationShip = true;
+		}
+
+		if (hasRelationShip) {
```

Comment:
You don't need to create a variable because of an IF condition.
You can simply check if the entity is not null.